### PR TITLE
Delete NCP created resources in cleanup 

### PR DESCRIPTION
**Pull request by yuntanghsu**
In this commit, we delete NCP created resource (share/sharedResource/certificate/lbAppProfile/lbPersistenProfile) in the cleanup process.

### CODE REVIEW

File: pkg/nsx/services/vpc/vpc.go
Comment (TaoZou1) [inline pkg/nsx/services/vpc/vpc.go:309]: certificate -> profile

File: pkg/nsx/services/vpc/vpc.go
Comment (TaoZou1) [inline pkg/nsx/services/vpc/vpc.go:272]: does it need store? just query and handling the result seems enough.

Comment (TaoZou1) [reply pkg/nsx/services/vpc/vpc.go:272]: using store could reuse the logic. looks fine.

File: pkg/nsx/services/vpc/vpc.go
Comment (yuntanghsu) [reply pkg/nsx/services/vpc/vpc.go]: Thanks for reviewing.
This func is unused. I forget to delete it.
Removed it. Thanks.

**Review by dantingl — DISMISSED** [review]
Yun-Tang, please rebase your patch

**Review by dantingl — DISMISSED** [review]

**Review by salv-orlando — APPROVED** [review]
Approving for triggering workflow run

**Review by salv-orlando — APPROVED** [review]

**Comment by salv-orlando** [comment]
> Approving for triggering workflow run

@yuntanghsu you need approval from a maintainer, unfortunately

**Review by dantingl — APPROVED** [review]

**Review by TaoZou1 — APPROVED** [review]

**Pull request merged**
